src/main/java/uk/gov/hmcts/reform/blobrouter/config/StorageConfiguration.java

```diff
@@ -15,4 +17,18 @@ public StorageSharedKeyCredential getStorageSharedKeyCredential(
     ) {
         return new StorageSharedKeyCredential(accountName, accountKey);
     }
+
+    @Bean("bulkscan-storage-client")
+    public static BlobServiceClient getBulkScanStorageClient(
+        @Value("storage.bulkscan.connection-string") String connectionString
```

Comment:
do we use connection string or storage key and account name for bulk scan storage ?
 or have decided to use connection string for both ?

Reply:
Oh, I forgot to mention. This PR is based on https://github.com/hmcts/blob-router-service/pull/90, where I'm replacing account name + key with connection string for crime, and for consistency also for bulk scan.